packages/rocketchat-ui/client/lib/textarea-autogrow.js

```diff
@@ -54,9 +56,9 @@
 				}
 
 				shadow.css('width', $self.width());
-				shadow.html(val + (noFlickerPad === 0 ? '...' : '')); // Append '...' to resize pre-emptively.
+				shadow.html(val); // Append '...' to resize pre-emptively.
```

Comment:
the comment does not make sense anymore

Reply:
your brain gets smart but your head gets dumb 🎶